dev/src/codewind/project/Project.ts

```diff
@@ -464,6 +466,77 @@ export default class Project implements vscode.QuickPickItem {
         });
     }
 
+    public async onLoadRunnerUpdate(event: {projectID: string, status: string, timestamp: string}): Promise<void> {
+        Log.d(`${this.name} load runner status changed to ${event.status}`, event);
+        switch (event.status) {
+            case "hcdReady":
+            case "profilingReady":
+                Log.d("Profiling data is ready to be saved to workspace");
+                const loadTestPath = path.join(this.localPath.fsPath, "load-test");
+                const timestampPath = path.join(loadTestPath, event.timestamp);
+                let fileName = "";
+                if (this.language === "java") {
+                    fileName = "profiling.hcd";
+                } else if (this.language === "nodejs") {
+                    fileName = "profiling.json";
+                }
+                const profilingOutPath = path.join(timestampPath, fileName);
+                try {
+                    await fs.promises.mkdir(timestampPath, {recursive: true});
```

Comment:
Have you tried this on windows? I am pretty sure recursive `mkdir` doesn't work there. https://github.com/nodejs/node/issues/24698

Reply:
Not yet tested on windows. 

Alternative would be making the folders individually, might be safer to use that approach regardless?

Reply:
Yes that's how I worked around it in the past, not so bad here since you're only creating 2 directories at most

Reply:
Updated. Had to remove the mkdir calls from the try/catch block to avoid `EEXIST` errors.